source/particle/property/crystal_preferred_orientation.cc

```diff
@@ -671,6 +671,8 @@ namespace aspect
             // within this if block and the next ones
             // Ordered vector where the first entry is the max/weakest and the last entry is the inactive slip system.
             std::array<unsigned int,4> indices;
+            for (auto &index: indices)
```

Comment:
You can zero initialize without a loop by using ``indices{};`` in the declaration.